igvm/cli.py

```diff
@@ -266,6 +266,11 @@ def parse_args():
         'vm_hostname',
         help='Hostname of the guest system',
     )
+    subparser.add_argument(
+        '--unretire',
+        action='store_true',
+        help='Unretire a VM, set it in maintenance mode',
+    )
```

Comment:
I think it would be easier to just have a target state switch and default one would be maintenance

Reply:
True, I'll implement it.

Reply:
It is implemented.